packages/remirror__extension-callout/src/callout-utils.ts

```diff
@@ -24,12 +24,17 @@ export const dataAttributeEmoji = 'data-callout-emoji';
 export function isValidCalloutAttributes(
   attributes: ProsemirrorAttributes,
 ): attributes is CalloutAttributes {
-  return !!(
-    attributes &&
-    isObject(attributes) &&
-    isString(attributes.type) &&
-    attributes.type.length > 0
-  );
+  if (attributes && isObject(attributes)) {
+    if (isString(attributes.type) && attributes.type.length > 0) {
+      return true;
```

Comment:
We `return true` before checking the emoji, so I think `isValidCalloutAttributes({ type: 'info', emoji: 97324 })` would return true, even thought the emoji is invalid?

Reply:
I think it allows us to just update the type value  without being wordily passing the emoji value(or just update emoji without mapping the existing type).

Reply:
Latter after this check, the existing value will be mapped in the attribute object if they are not provided. https://github.com/Gaaaga/remirror/blob/emojicallout-fix/packages/remirror__extension-callout/src/callout-utils.ts#L62

Reply:
But I think the point still stands, `isValidCalloutAttributes({ type: 'info', emoji: 97324 })` will return `true` when surely it should return `false`?

Reply:
You are right! let me do some changes here.